MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include test/*.py
```

Comment:
Fair enough. The only issue is that distutils/setuptools picks up test/test_*.py by default, so I just figured I'd include the rest. I can rename the directory to 'tests' instead so it doesn't pick up any of it.